<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Export</class>
 <widget class="QWidget" name="Export">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>741</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CinemaSimi - Exportar</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/recursos/upc_logo.png</normaloff>:/icons/recursos/upc_logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 234, 212);

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="encabezado">
     <item>
      <widget class="QLabel" name="lblLogo">
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons.qrc">:/icons/recursos/cinemaSimi.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblUPC">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="icons.qrc">:/icons/recursos/upc_logo.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="layoutPrincipal">
     <item>
      <widget class="QVideoWidget" name="VideoPlayer" native="true">
       <property name="minimumSize">
        <size>
         <width>720</width>
         <height>440</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(224, 230, 237);</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutSecundario_">
       <item>
        <layout class="QVBoxLayout" name="layoutOpciones">
         <item>
          <layout class="QHBoxLayout" name="layoutBtns">
           <item>
            <widget class="QPushButton" name="btnPlay">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">/*BUTTON */

QPushButton{

	background-color: rgb(132, 146, 166);
	color: white;
	border-width: 2px;
	border-radius: 4px;
	font: bold ;
}

QPushButton::hover{
background-color: rgb(73, 136, 175);
}

QPushButton::disabled{
	background-color:#B5B5B5;
	color: #D6D6D6;
	border-width: 2px;
	border-radius: 4px;
	font: bold ;
}

QPushButton::pressed {  
background-color: rgb(58, 113, 154) 
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/recursos/play_ic.png</normaloff>:/icons/recursos/play_ic.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnPause">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">/*BUTTON */

QPushButton{

	background-color: rgb(132, 146, 166);
	color: white;
	border-width: 2px;
	border-radius: 4px;
	font: bold ;
}

QPushButton::hover{
background-color: rgb(73, 136, 175);
}

QPushButton::disabled{
	background-color:#B5B5B5;
	color: #D6D6D6;
	border-width: 2px;
	border-radius: 4px;
	font: bold ;
}

QPushButton::pressed {  
background-color: rgb(58, 113, 154) 
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/recursos/pause_ic.png</normaloff>:/icons/recursos/pause_ic.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnStop">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">/*BUTTON */

QPushButton{

	background-color: rgb(132, 146, 166);
	color: white;
	border-width: 2px;
	border-radius: 4px;
	font: bold ;
}

QPushButton::hover{
background-color: rgb(73, 136, 175);
}

QPushButton::disabled{
	background-color:#B5B5B5;
	color: #D6D6D6;
	border-width: 2px;
	border-radius: 4px;
	font: bold ;
}

QPushButton::pressed {  
background-color: rgb(58, 113, 154) 
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/recursos/stop_ic.png</normaloff>:/icons/recursos/stop_ic.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QProgressBar" name="progressBar">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>400</width>
               <height>10</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>13</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="styleSheet">
              <string notr="true">QProgressBar
{
	border: 0.5px;

	border-color: #8492A6;
	border-radius: 9px;
	background-color: #ffffff;
}
QProgressBar::chunk 
{
	background-color: #94A5C9;
}   </string>
             </property>
             <property name="value">
              <number>24</number>
             </property>
             <property name="textVisible">
              <bool>false</bool>
             </property>
             <property name="invertedAppearance">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btnCancel">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>180</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">/*BUTTON */

QPushButton{

	background-color:rgb(132, 146, 166);
	color: white;
	border-width: 2px;
	border-radius: 10px;
	font: bold ;
}

QPushButton::disabled{
	background-color:#B5B5B5;
	color: #D6D6D6;
	border-width: 2px;
	border-radius: 10px;
	font: bold ;
}

QPushButton::hover{
background-color: rgb(73, 136, 175);
}

QPushButton::pressed {  
background-color: rgb(58, 113, 154) 
}
</string>
             </property>
             <property name="text">
              <string>Cancelar Exportación</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="lblLocation">
             <property name="minimumSize">
              <size>
               <width>350</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
             <property name="text">
              <string>C:/User/Videos/</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-image: url(:/icons/recursos/upload.png);
QPushButton{
	border-width: 2px;
	border-radius: 4px;
}</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="icons.qrc">
               <normaloff>:/icons/recursos/download.png</normaloff>:/icons/recursos/download.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>25</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lblLenght">
             <property name="text">
              <string>Duración del vídeo:  00:01:20</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lblTipe">
             <property name="text">
              <string>Tipo de vídeo:         .mp4</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLabel" name="lblSize">
             <property name="text">
              <string> Tamaño del video: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QComboBox" name="comboBox">
             <property name="styleSheet">
              <string notr="true">/*style for the QComboBox*/

#comboBox{
	border: 1px solid rgb(132, 146, 166);
	border-radius: 1px;
	padding-left: 10px;
	background-color: 	rgb(255,255,255);
	height: 28px;
}

/*style for dropdown area*/

#comboBox::drop-down{
	border: 0px;
}

/*style for drop down arrow*/
#comboBox::down-arrow{
	image: url(:/icons/recursos/down.png);
	width: 12px;
	height: 12px;
	margin-right: 15px;
}

/*style for QComboBox after open select menu*/

#comboBox::on{
	border: 2px solid #c2dbfe;
}

/*style for list menu*/

#comboBox QListView{
	font-size: 12px;
	border: 1px solid rgba(0,0,0,10%);
	padding: 5px;
	background-color: 	rgb(255,255,255);
	outline: 0px;
}


/*style for list items*/
 
#comboBox QListView:item{
	padding-left: 10px;
	background-color: 	rgb(255,255,255);
}

#comboBox QListView:item:hover{
	background-color:  rgb(73, 136, 175);
}

#comboBox QListView:item:selected{
	background-color: rgb(73, 136, 175);
}


</string>
             </property>
             <item>
              <property name="text">
               <string>Seleccione la calidad de video</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>360p</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>480p</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>720p</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>1080p</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLabel" name="lblEstimated">
             <property name="text">
              <string> Tiempo estimado:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnExportar">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">/*BUTTON */

QPushButton{

	background-color: rgb(132, 146, 166);
	color: white;
	border-width: 2px;
	border-radius: 10px;
	font: bold ;
}

QPushButton::disabled{
	background-color:#B5B5B5;
	color: #D6D6D6;
	border-width: 2px;
	border-radius: 10px;
	font: bold ;
}


QPushButton::hover{
background-color: rgb(73, 136, 175);
}

QPushButton::pressed {  
background-color: rgb(58, 113, 154) 
}
</string>
         </property>
         <property name="text">
          <string>Exportar</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
